apps/extension/src/state/password.ts

```diff
@@ -6,19 +6,24 @@ import { repeatedHash } from 'penumbra-crypto-ts';
 
 export interface PasswordSlice {
   hashedPassword: string | undefined;
+  isCorrectPassword: boolean;
   setPassword: (password: string) => string;
+  setCorrectPassword: () => void;
   clearPassword: () => void;
   isPassword: (password: string) => Promise<boolean>;
+  clearSessionPassword: () => void;
+  isUnlock: (password: string) => Promise<boolean>;
 }
 
 export const createPasswordSlice =
   (
     session: ExtensionStorage<SessionStorageState>,
     local: ExtensionStorage<LocalStorageState>,
   ): SliceCreator<PasswordSlice> =>
-  set => {
+  (set, get) => {
     return {
       hashedPassword: undefined,
+      isCorrectPassword: true,
```

Comment:
Think this is risky having this default true. The component itself should just have;

`const result = isPassword(password)`. And the rest of the UI can react to that locally there.

Reply:
I would remove `setCorrectPassword: () => void;`

Reply:
I don't understand! you wrote in the previous version that I should put it in slice